ExampleBlazor/ExampleBlazor.csproj

```
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+    <PropertyGroup>
+        <TargetFramework>net6.0</TargetFramework>
+        <Nullable>enable</Nullable>
+        <ImplicitUsings>enable</ImplicitUsings>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <PackageReference Include="InfluxDB.Client" Version="4.4.0-dev.6843" />
```

The package should be referenced via relative path.

```suggestion
      <ProjectReference Include="..\Client\Client.csproj" />
```